### ISSUE TITLE
multiple language spell checking/correction

### ISSUE THREAD

**Issue by Silther** [cinnyapp/cinny-desktop#196]
### Describe the problem

I use this app both with German and English people, so I set Windows to German and English, but the app currently only support English or one of the languages installed in Windows.

### Describe the solution you'd like

Please add support for multiple languages (manually select them in the app or better use the languages of the operating system.

**Comment by squid-f** [comment]
Hi. Does Cinndy really have spell check capability? I don't see any done when I type.
It would be awesome if spell check works.

**Comment by loufidelity** [comment]
> Hi. Does Cinndy really have spell check capability? I don't see any done when I type. It would be awesome if spell check works.

Yes it has. I am also using Cinny with a second language, and everything I type has the red dotted underline marking it as a typo.

In my case I would like to turn the spell checking off completely, but I cannot find any option in Cinny to do so.
